src/tests/ftest/util/soak_utils.py

```diff
@@ -459,6 +460,119 @@ def launch_vmd_identify_check(self, name, results, args):
     self.log.info("<<<PASS %s: %s completed at %s>>>\n", self.loop, name, time.ctime())
 
 
+def launch_reboot(self, pools, name, results, args):
+    """Execute server unexpected reboot.
+
```

Comment:
It will be nice to have some comments on the test steps involved here... Otherwise, we have to go through the code to understand what is happening during this testing.